tests/regression/test_zero_forms.py

```diff
@@ -10,43 +10,38 @@ def mesh(request):
     return UnitSquareMesh(10, 10, quadrilateral=quadrilateral)
 
 
-@pytest.fixture(scope='module')
-def one(mesh):
-    return Constant(1, domain=mesh)
-
-
 domains = [(1, 2),
            (2, 3),
            (3, 4),
            (4, 1),
            (1, 2, 3, 4)]
 
 
-def test_ds_dx(one):
-    assert np.allclose(assemble(one*dx + one*ds), 5.0)
+def test_ds_dx(mesh):
+    assert np.allclose(assemble(1*dx(domain=mesh) + 1*ds(domain=mesh)), 5.0)
```

Comment:
I'm not sure that this is the right thing to do. We still want tests with `Constant` to work here. Doesn't getting rid of `domain=mesh` and passing it to the measure work even with constants?